bit_stream_filter_context_test.go

```diff
@@ -0,0 +1,34 @@
+package astiav
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestBitStreamFilterContext(t *testing.T) {
+	bsf := FindBitStreamFilterByName("null")
+	require.NotNil(t, bsf)
+
+	bsfc, err := AllocBitStreamFilterContext(bsf)
+	require.NotNil(t, bsfc)
+	require.NoError(t, err)
+	defer bsfc.Free()
+
+	cl := bsfc.Class()
+	require.NotNil(t, cl)
+	require.Equal(t, "AVBSFContext", cl.Name())
+
+	bsfc.SetTimeBaseIn(NewRational(15, 1))
+	require.Equal(t, NewRational(15, 1), bsfc.TimeBaseIn())
+
+	cp1 := AllocCodecParameters()
+	require.NotNil(t, cp1)
+	defer cp1.Free()
```

Comment:
OK so I figured out why tests were failing on Windows, basically you need to remove the `defer cp1.Free()` since the context seems to be taking ownership of the codec parameters and free it when calling `bsfc.Free()`

Reply:
thanks, great catch! :)

Reply:
probably calling free twice on unix didn't raise an error while on windows it'll